src/screens/Settings/DeerSettings.tsx

```diff
@@ -499,6 +506,17 @@ export function DeerSettingsScreen({}: Props) {
               </Toggle.LabelText>
               <Toggle.Platform />
             </Toggle.Item>
+            <Toggle.Item
+              name="high_quality_images"
+              label={_(msg`Display images in higher quality`)}
+              value={highQualityImages}
+              onChange={value => setHighQualityImages(value)}
+              style={[a.w_full]}>
+              <Toggle.LabelText style={[a.flex_1]}>
+                <Trans>Display images in higher quality</Trans>
```

Comment:
could be slightly more descriptive and say something about JPEG vs PNG encoding? not needed but our userbase is technical enough to appreciate the difference...
"rewrite bsky cdn images to use png instead of jpeg"

your language is fine... and shorter. feel free to ignore this, its a nit

Reply:
I considered it (and tried to explain it better before settling on my current wording), but the UI size unfortunately requires the title to be pretty concise. I wonder if it'd be worth adding a little aside or tooltip?

Reply:
up to you, either way is fine

Reply:
Added a `Images will be served as PNG instead of JPEG. Images will take longer to load and use more bandwidth.` line.